[2025-10-20T16:44:05.188+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-10-20T16:44:05.210+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: Airflow_Lab1_Rithika.build_save_model_task manual__2025-10-20T16:43:58.138453+00:00 [queued]>
[2025-10-20T16:44:05.220+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: Airflow_Lab1_Rithika.build_save_model_task manual__2025-10-20T16:43:58.138453+00:00 [queued]>
[2025-10-20T16:44:05.221+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 1
[2025-10-20T16:44:05.235+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): build_save_model_task> on 2025-10-20 16:43:58.138453+00:00
[2025-10-20T16:44:05.248+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=599) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-10-20T16:44:05.251+0000] {standard_task_runner.py:63} INFO - Started process 618 to run task
[2025-10-20T16:44:05.250+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'Airflow_Lab1_Rithika', 'build_save_model_task', 'manual__2025-10-20T16:43:58.138453+00:00', '--job-id', '31', '--raw', '--subdir', 'DAGS_FOLDER/***_lab1_dag.py', '--cfg-path', '/tmp/tmp7vdjby5v']
[2025-10-20T16:44:05.253+0000] {standard_task_runner.py:91} INFO - Job 31: Subtask build_save_model_task
[2025-10-20T16:44:05.330+0000] {task_command.py:426} INFO - Running <TaskInstance: Airflow_Lab1_Rithika.build_save_model_task manual__2025-10-20T16:43:58.138453+00:00 [running]> on host bddbc10ace4f
[2025-10-20T16:44:05.453+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='rithika' AIRFLOW_CTX_DAG_ID='Airflow_Lab1_Rithika' AIRFLOW_CTX_TASK_ID='build_save_model_task' AIRFLOW_CTX_EXECUTION_DATE='2025-10-20T16:43:58.138453+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-10-20T16:43:58.138453+00:00'
[2025-10-20T16:44:05.455+0000] {taskinstance.py:430} INFO - ::endgroup::
[2025-10-20T16:44:05.639+0000] {logging_mixin.py:188} INFO - âœ… Model built and saved successfully at: /opt/***/working_data/model_rithika.pkl
[2025-10-20T16:44:05.640+0000] {python.py:237} INFO - Done. Returned value was: /opt/***/working_data/model_rithika.pkl
[2025-10-20T16:44:05.641+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-10-20T16:44:05.674+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=Airflow_Lab1_Rithika, task_id=build_save_model_task, run_id=manual__2025-10-20T16:43:58.138453+00:00, execution_date=20251020T164358, start_date=20251020T164405, end_date=20251020T164405
[2025-10-20T16:44:05.712+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2025-10-20T16:44:05.731+0000] {taskinstance.py:3503} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-10-20T16:44:05.735+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
